components/function-controller/cmd/jobinit/main.go

```diff
@@ -0,0 +1,39 @@
+package main
```

Comment:
Hm... do we need it? Why not just a simple image with a git client and simple bash logic?